client/my-sites/plans/ecommerce-trial/wooexpress-plans/style.scss

```diff
@@ -28,7 +28,53 @@
 		font-family: $sans;
 	}
 
-	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price {
+	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price,
+	.plan-features-2023-grid__plan-comparison-grid-container .plan-comparison-grid .plan-price {
 		font-family: $sans;
 	}
+
+	.plan-price .plan-price__currency-symbol {
+		font-size: 1.5rem;
```

Comment:
Just a suggestion. We have font-size variables defined here. You can also use it. 🙂 

https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans/woo-express-plans-page/style.scss

https://github.com/Automattic/wp-calypso/blob/1b09615ebc1e80cc55e6507fa166bc551cab2cad/packages/typography/styles/woo-commerce.scss#L3-L15

Reply:
Great suggestion; updated the CSS to use these variables!